tests/e2e/framework/kubernetes.go

```diff
@@ -1025,6 +1031,31 @@ EOF`, k.KubeConfig, dummyValidationRule)
 	return nil
 }
 
+func (k *KubeInfo) waitForIstioOperator() error {
+
+	get := fmt.Sprintf(`cat << EOF | kubectl --kubeconfig=%s get icp example-istiocontrolplane -n istio-operator
+-o yaml
```

Comment:
really confused what this is doing?

Reply:
run `kubectl get icp example-istiocontrolplane -n istio-operator -o yaml` to make sure we have `Status` field updated for icp -- istio operator will set the status filed only if the reconcile fininshed.

Reply:
but what do we have this EOF cat stuff?

Reply:
removed EOF and cat stuff